### ISSUE TITLE
error: Found argument 'create' which wasn't expected

### ISSUE THREAD

**Issue by rmsc** [Mustafif/texcreate_old#1]
The 'create' subcommand seems to have been removed:

```bash
$ texcreate create
error: Found argument 'create' which wasn't expected, or isn't valid in this context

USAGE:
    texcreate <SUBCOMMAND>

For more information try --help
```

Not on the --help documentation either:

```bash
$ texcreate --help
texcreate 1.2.0
Create LaTeX projects using prebuilt templates

USAGE:
    texcreate <SUBCOMMAND>

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

SUBCOMMANDS:
    build     Build a project using a config.toml file
    doc       Opens the TexCreate documentation
    edit      Quickly change config.toml parameters
    help      Prints this message or the help of the given subcommand(s)
    init      Initialize a config.toml file
    list      Lists all the available templates
    update    Updates to latest version
    web       Opens the TexCreate web version
```

I was going to try to pinpoint where that happened in git history, but the commit messages are all the same ("Update")... :)

**Comment by Mustafif** [comment]
oh my sorry about that, so in v1.1.0, I removed a command called `create` which was a one command `config.toml`. The article I wrote of the changes can be found [here](https://dev.to/mustafif/texcreate-v110-4898). I'll try doing more in changelog, and be better in commit messages. Here is also an article I wrote today of new changes  being added in [v2.0](https://dev.to/mustafif/texcreate-v20-announcement-8dg)
